pkg/cluster/cluster.go

```diff
@@ -21,9 +21,18 @@ import (
 	"regexp"
 	"strings"
 
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 )
 
+type AddOn struct {
```

Comment:
We should call this `AddOnItem` or `AddOnStatus` or some other name that doesn't get confused with `cmv1.AddOn`